src/schema/search.ts

```diff
@@ -312,6 +337,28 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
         query: args.query,
       };
     },
+    searchTagSuggestions: async (
+      source,
+      { query, limit }: SearchSuggestionArgs,
+      ctx,
+    ): Promise<GQLSearchSuggestionsResults> => {
+      const searchQuery = ctx.con
+        .getRepository(Keyword)
+        .createQueryBuilder()
+        .select(`value as id, COALESCE(flags->>'title', value) as title`)
```

Comment:
What is this title thingy?

Reply:
I think its the flag that nimrod can set for sep purpose.
I think we should just search based on value (i think its called)

Reply:
thanks for the explanation